app/assets/v2/js/pages/fulfill_bounty.js

```diff
@@ -0,0 +1,120 @@
+
+window.onload = function(){
+    //a little time for web3 injection
+    setTimeout(function(){
+        var account = web3.eth.accounts[0];
+
+        if (typeof localStorage['githubUsername'] !='undefined'){
+            $('input[name=githubUsername]').val(localStorage['githubUsername']);
+        }
+        if (typeof localStorage['notificationEmail'] !='undefined'){
+            $('input[name=notificationEmail]').val(localStorage['notificationEmail']);
+        }
+        if (typeof localStorage['acceptTOS'] !='undefined' && localStorage['acceptTOS']){
+            $('input[name=terms]').attr('checked','checked');
+        }
+        if(getParam('source')){
+            $('input[name=issueURL]').val(getParam('source'));
+        }
+
+
+        $('#submitBounty').click(function(e){
+            mixpanel.track("Claim Bounty Clicked", {});
+            loading_button($('#submitBounty'));
+            e.preventDefault();
+            var notificationEmail = $('input[name=notificationEmail]').val();
+            var githubUsername = $('input[name=githubUsername]').val();
+            var issueURL = $('input[name=issueURL]').val();
+            var claimee_metadata = JSON.stringify({
+                notificationEmail : notificationEmail,
+                githubUsername : githubUsername,
+            });
+            localStorage['githubUsername'] = githubUsername;
+
+            var isError = false;
+            if($('#terms:checked').length == 0){
+                _alert({ message: "Please accept the terms of service." });
+                isError = true;
+            } else {
+                localStorage['acceptTOS'] = true;
+            }
+            if(issueURL == ''){
+                _alert({ message: "Please enter a issue URL." });
+                isError = true;
+            }
+            if(isError){
+                unloading_button($('#submitBounty'));
+                return;
+            }
+
+            var bounty = web3.eth.contract(bounty_abi).at(bounty_address());
+
+            ipfs.ipfsApi = IpfsApi({host: 'ipfs.infura.io', port: '5001', protocol: "https", root:'/api/v0'});
+            ipfs.setProvider({ host: 'ipfs.infura.io', port: 5001, protocol: 'https', root:'/api/v0'});
+
+        // ipfs.addJSON({description: data, sourceFileName: this.state.sourceFileName, sourceDirectoryHash: this.state.sourceDirectoryHash, sourceFileHash: this.state.sourceFileHash, contact: contact}, (err, succ)=> {
+
+            var submit = {
+                description: issueURL,
+                sourceFileName: "",
+                sourceFileHash: "",
+                sourceDirectoryHash: "",
+                contact: notificationEmail,
```

Comment:
should we be storing information that's missing from the database (like the githubusername, expereience level required... etc) in this JSON blob?  its got to go somewhere..

Reply:
Not here I don't think, but yes it should be in the database.  Looking into that one..

Reply:
whered we land on this?  do we need to track it as an ongoing issue??

Reply:
There was no issue here.  I'm confident it was just a matter of the fields not being filled out in the bounty submission.